### DISCUSSION TITLE
OAI LTE with IMS does not work

### DISCUSSION THREAD

**Discussion by altis82** [herlesupreeth/docker_open5gs#254] [General]
Hello
I have tried to run OAI enB in our project. I can connect 2 phones but I could not make a call between them. When I send message from sim1 to sim2, it also has errors although It receives the message at the end.
I would like to attach the wireshark file here. Thanks

https://shorturl.at/norCR

**Comment by herlesupreeth** [comment]
I see SIP registration of only one UE and not the second UE. Can you please perform the following steps and send me the pcap?

1. Stop all containers
2. Start the wireshark trace
3. Bring up EPC + IMS 
4. Start eNB and connect with EPC
5. Attach two UE one after another
6. Attempt a call
7. Stop the trace and attach here in discussion
